src/Parallax.js

```diff
@@ -11,7 +11,7 @@ class Parallax extends Component {
     return (
       <div className={cx('parallax-container', className)}>
         <div className='parallax'>
-          <img src={imageSrc}/>
+          <img src={imageSrc} />
```

Comment:
Allright. Shall I make a small PR for this? Cause it needs to be fixed for all tests to pass I guess :)